### ISSUE TITLE
SME Interviews

### ISSUE THREAD

**Issue by RenataKeck** [department-of-veterans-affairs/va.gov-team#69498]
- [ ] Identify other teams to talk with about related work
- [ ] Prep question set
- [ ] Schedule and run interviews

**Comment by uxrebecca** [comment]
Could talk to Lighthouse team (they've built tools reps use to submit claims), Benefits Claims researchers, Office of General Counsel (manage accreditation - understand workflows of reps), Rep Mgmt Team (they're building other half of this - claimant facing side). Alex can help!

**Comment by uxrebecca** [comment]
Other groups:
VSOs themselves. 4 categories:
1. The big VSOs (DAVs, VFWs) - probably pretty well organized, have existing tools they use, processes.
2. Smaller VSOs - someone from OGC could help us get more granular. 
3. Attorneys that represent veterans
4. Claims agents

**Comment by uxrebecca** [comment]
For Lighthouse, Derrick is on that team. Kayla Watanabe a is Derrick's PM. 
Benefits Claims researchers - can ask Kayla about that too. 
OGC - could ask Lesley. Might be more of a Slack thing than an interview.

**Comment by RivkaCGates** [comment]
@uxrebecca do we have this documented anywhere but this ticket? If so, can we add a link please?

**Comment by uxrebecca** [comment]
Yes, this list is now documented in the [research plan](https://docs.google.com/document/d/1i7SGMj0ishpbNZ1tRYY_ZKZL_I9hHiQm6JXUnwT-oT4/edit) (also linked from #70297)
